frontend/src/components/authentication/RegistrationForm.jsx

```diff
@@ -230,28 +334,50 @@ class RegistrationForm extends Component {
             document.getElementById("email-address-field").focus();
             // account successfully created
           } else {
-            this.setState({ showDialog: true, accountExistsError: false });
+            this.setState({ showCreatedAccountDialog: true, accountExistsError: false });
           }
         });
     }
     event.preventDefault();
   };
 
+  showPrivacyNoticePopup = event => {
+    this.setState({ showPrivacyNoticeDialog: true });
+    event.preventDefault();
```

Comment:
Can you add a comment explaining why this is important?

Reply:
Comments added.
See commit #https://github.com/code-for-canada/project-thundercat/pull/330/commits/a7433527072d031e5736025cb508bf7e71cf8154